src/containers/Home.tsx

```diff
@@ -19,6 +19,7 @@ interface StateProps {
 
 interface DispatchProps {
   onDeleteRecordings: (recordingIds: string[]) => void
+  onDeleteSortings: (sortingIds: string[]) => void
```

Comment:
This now needs to be passed in explicitly into WorkspaceView, because WorkspaceView no longer has access to redux dispatch.